### PR TITLE
Change filters and list view for DocType LANDA member (LANDA-699)

### PR DESCRIPTION
**Pull request by melanieroedel**
Switch from Full Name back to First Name and Last Name separately to ensure the list view can be sorted by one or the other.
Correcting the columns of list view that weren't displayed anyway.

### CODE REVIEW

**Comment by melanieroedel** [comment]
@barredterra Can we merge this individually as a minor change or should it be included in a release?

**Comment by barredterra** [comment]
This PR can be merged independently of any release. We always make a release when merging `version-13-hotfix` into `version-13`.

**Comment by melanieroedel** [comment]
Sorry, wasn't clear. Can we merge this into `version-13` right after merging into `version-13-hotfix`?

**Review by barredterra — APPROVED** [review]

**Review by barredterra — APPROVED** [review]

**Pull request merged**
